src/Errors.ts

```diff
@@ -1,57 +1,12 @@
-interface ErrorInfoValues {
```

Comment:
@ttypic can we have these changes in a separate commit please?

Reply:
yes, sure, locally can't build library without this (will investigate why)

Reply:
not a dealbreaker if you can't, though would be good to understand why that is

Reply:
problem was, because I've updated ably-js to `1.2.45`, rollbacked it